media-sound/strawberry/strawberry-9999.ebuild

```diff
@@ -19,98 +17,79 @@ fi
 
 LICENSE="GPL-3"
 SLOT="0"
-IUSE="cdda debug +gstreamer ipod moodbar mtp pulseaudio qt6 soup +udisks vlc"
+IUSE="alsa cdda chromaprint dbus debug kde +loudness ipod moodbar mtp +pulseaudio streaming +udisks X"
 
 BDEPEND="
 	sys-devel/gettext
 	virtual/pkgconfig
-	!qt6? ( dev-qt/linguist-tools:5 )
 "
+
+#INFO: alsa-lib is always required in linux even if its not built
 COMMON_DEPEND="
 	dev-db/sqlite:=
 	dev-libs/glib:2
 	dev-libs/icu:=
-	dev-libs/protobuf:=
 	media-libs/alsa-lib
 	media-libs/taglib
-	!qt6? (
-		dev-qt/qtconcurrent:5
-		dev-qt/qtcore:5
-		dev-qt/qtdbus:5
-		dev-qt/qtgui:5
-		dev-qt/qtnetwork:5[ssl]
-		dev-qt/qtsql:5[sqlite]
-		dev-qt/qtwidgets:5
-		dev-qt/qtx11extras:5
-		x11-libs/libX11
-	)
-	qt6? (
-		dev-libs/kdsingleapplication[qt6(+)]
-		dev-qt/qtbase:6[concurrent,dbus,gui,network,ssl,sql,sqlite,widgets]
-	)
+	dev-libs/kdsingleapplication[qt6(+)]
+	dev-qt/qtbase:6[concurrent,dbus?,gui,network,ssl,sql,sqlite,widgets,X?]
+	media-libs/gstreamer:1.0
+	media-libs/gst-plugins-base:1.0
 	cdda? ( dev-libs/libcdio:= )
-	gstreamer? (
-		media-libs/chromaprint:=
-		media-libs/gstreamer:1.0
-		media-libs/gst-plugins-base:1.0
+	chromaprint? ( media-libs/chromaprint:= )
+	ipod? (
+		media-libs/libgpod
+		x11-libs/gdk-pixbuf
 	)
-	ipod? ( media-libs/libgpod )
 	moodbar? ( sci-libs/fftw:3.0 )
 	mtp? ( media-libs/libmtp )
+	loudness? ( media-libs/libebur128 )
 	pulseaudio? ( media-libs/libpulse )
-	vlc? ( media-video/vlc )
 "
 # Note: sqlite driver of dev-qt/qtsql is bundled, so no sqlite use is required; check if this can be overcome someway;
 RDEPEND="${COMMON_DEPEND}
-	gstreamer? (
-		media-plugins/gst-plugins-meta:1.0
-		soup? ( media-plugins/gst-plugins-soup:1.0 )
-		media-plugins/gst-plugins-taglib:1.0
-	)
-	mtp? ( gnome-base/gvfs[mtp] )
+	media-plugins/gst-plugins-meta:1.0[taglib]
```

Comment:
IMO it is an unfortunate change to require `gst-plugins-meta[taglib]` instead of requiring both gst-plugins-meta and gst-plugins-taglib.

The current state of affairs means that users by default get USE flag conflicts. Furthermore, uninstalling strawberry will leave behind a package.use entry for gst-plugins-meta that may not end up getting cleaned up, preventing taglib from being naturally depcleaned along with strawberry.

This jammed up the official binhost once strawberry 1.2.2 was stabilized.